app/views/admin/AdministerUserModal.coffee

```diff
@@ -52,20 +53,53 @@ module.exports = class AdministerUserModal extends ModalView
     @trialRequest = @trialRequests.first()
     super()
     
+  onClickCreatePayment: ->
+    service = $('#payment-service').val()
+    amount = parseInt($('#payment-amount').val())
+    amount = 0 if isNaN(amount)
+    gems = parseInt($('#payment-gems').val())
```

Comment:
These should use `@$` instead of `$`.

Reply:
Why?

Reply:
That way actions don't accidentally affect something in the global DOM. Also, it's generally better if we ever test something, because it's not necessarily the case that the Backbone View you're testing has been mounted.